plugin/core/state.py

```
@@ -26,19 +75,761 @@ def __init__(self):
+                    # multiprocessing.Event is backed by a shared semaphore/pipe and works across fork.
+                    self._plugin_response_notify_event = multiprocessing.Event()
+        return self._plugin_response_notify_event
+
+    def _get_or_create_response_event(self, request_id: str):
+        rid = str(request_id)
+        # Force init of shared manager + maps (important: do not create a new Manager per process)
+        _ = self.plugin_response_map
+        try:
+            event_map = self.plugin_response_event_map
+            ev = event_map.get(rid)
+        except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)